setup.cfg

```diff
@@ -49,7 +49,7 @@ single-quotes = true
 
 [tool:pytest]
 minversion = 5.4
-addopts = -p no:doctest --xdoctest --xdoctest-style=google --random-order --random-order-bucket=global
+addopts = -v -p no:doctest --xdoctest --xdoctest-style=google --random-order --random-order-bucket=global --cov=app --cov-report html
```

Comment:
Add the coverage args as `--cov=./ --cov-report=html` to mirror the automated CI testing and coverage calculation